isacc_messaging/api/isacc_record_creator.py

```diff
@@ -236,8 +236,8 @@ def generate_incoming_message(self, message, time: datetime = None, patient: Pat
         if priority is None:
             priority = "routine"
 
-        if patient is None:
-            return "Need patient"
+        if patient is None or patient.active != True:
+            return "Need active patient"
```

Comment:
i realize this is just a change from before, but this is close to an unnecessary check.  if we're going to check for a required parameter and it isn't there or valid, please raise an exception.